settings.d/openscap.yml.example

```diff
@@ -22,3 +22,7 @@
 # Directory where OpenSCAP report XML are stored
 # In case sending to Foreman succeeded, yet failed to save to reportsdir
 #:failed_dir: /usr/share/foreman-proxy/openscap/failed
+
+# Directory where corrupted OpenSCAP report XML are stored
+# When proxy cannot parse the report sent by client
+#:corrupted_dir: /usr/share/foreman-proxy/openscap/corrupted
```

Comment:
I guess this will need a change in installer [here](https://github.com/theforeman/puppet-foreman_proxy/blob/master/manifests/plugin/openscap/params.pp#L18) and [here](https://github.com/theforeman/puppet-foreman_proxy/blob/master/templates/plugin/openscap.yml.erb), right? Could you prepare PR for it pls?